ocs_ci/utility/utils.py

```diff
@@ -1832,6 +1832,24 @@ def get_ocp_repo(rhel_major_version=None):
     return path
 
 
+def get_running_acm_version():
+    """
+    Get the currently deployed ACM version
+
+    Returns:
+        string: ACM version
+
+    """
+    occmd = "oc get mch multiclusterhub -n open-cluster-management -o json"
```

Comment:
this can also be replaced with `oc get mch multiclusterhub -n open-cluster-management -o jsonpath='{.status.currentVersion}'` so that we dont need to split

Reply:
$ oc get mch multiclusterhub -n open-cluster-management -o jsonpath='{.status.currentVersion}'
2.6.1